electron-react/src/pages/RateLimitedWallet.js

```diff
@@ -0,0 +1,974 @@
+                  label="User Pubkey"
+                  value={pubkey}
+                  variant="outlined"
+                />
+              </Box>
+              <Box>
+                <Button
+                  onClick={copy}
+                  className={classes.copyButton}
+                  variant="contained"
+                  color="secondary"
+                  disableElevation
+                >
+                  Copy
+                </Button>
+              </Box>
+            </Box>
+          </div>
+        </Grid>
+        <Grid item xs={12}>
+          <div className={classes.setupSection}>
+            <Box display="flex">
+              <Box flexGrow={1} style={{ marginTop: 10, marginBottom: 0 }}>
+                <Typography variant="subtitle1">When you receive the setup info packet from your admin, enter it below to complete your Rate Limited Wallet setup:</Typography>
+              </Box>
+            </Box>
+            <Box display="flex">
+              <Box flexGrow={1} style={{ marginTop: 0 }}>
+                <TextField
+                  id="filled-secondary"
+                  variant="filled"
+                  color="secondary"
+                  fullWidth
+                  inputRef={input => {
+                    ip_input = input;
+                  }}
+                  margin="normal"
+                  label="Info Packet"
+                />
+              </Box>
+            </Box>
+          </div>
+          <div className={classes.setupSection}>
+            <Box display="flex">
+              <Box>
+                <Button
+                  onClick={submit}
+                  className={classes.submitButton}
+                  variant="contained"
+                  color="primary"
+                >
+                  Submit
+                </Button>
+              </Box>
+            </Box>
+          </div>
+        </Grid>
+      </Grid>
+    </Paper>
+  );
+};
+
+const RLDetailsCard = props => {
+  var id = props.wallet_id;
+
+  const name = useSelector(state => state.wallet_state.wallets[id].name);
+  const data = useSelector(state => state.wallet_state.wallets[id].data);
+  const data_parsed = JSON.parse(data);
+  const user_pubkey = data_parsed["user_pubkey"]
+  const admin_pubkey = data_parsed["admin_pubkey"]
+  const interval = data_parsed["interval"]
+  const limit = data_parsed["limit"]
+  const origin = data_parsed["rl_origin"]
+  const origin_string = JSON.stringify(origin)
+  const infopacket = { "interval": interval, "limit": limit, "origin_string": origin_string, "admin_pubkey": admin_pubkey }
+
+  const ip_string = JSON.stringify(infopacket)
+  const ip_buf = Buffer.from(ip_string, 'utf8');
+  const ip_hex = ip_buf.toString('hex');
+
+  function user_copy() {
+    navigator.clipboard.writeText(user_pubkey);
+  }
+
+  function ip_hex_copy() {
+    navigator.clipboard.writeText(ip_hex);
+  }
+
+  const classes = useStyles();
+  if (name === "RL User") {
```

Comment:
@lipalong wallet name is not guaranteed to be "RL User", use type from rl_info.type
rl_info.type is guaranteed to be "admin" | "user"